src/omv/modules/py_ml.c

```diff
@@ -194,25 +240,25 @@ static void py_ml_model_attr(mp_obj_t self_in, qstr attr, mp_obj_t *dest) {
                 dest[0] = MP_OBJ_FROM_PTR(self->input_shape);
                 break;
             case MP_QSTR_input_dtype:
-                dest[0] = mp_obj_new_str(&self->input_dtype, 1);
+                dest[0] = MP_OBJ_FROM_PTR(self->input_dtype);
                 break;
             case MP_QSTR_input_scale:
-                dest[0] = mp_obj_new_float(self->input_scale);
+                dest[0] = MP_OBJ_FROM_PTR(self->input_scale);
                 break;
             case MP_QSTR_input_zero_point:
-                dest[0] = mp_obj_new_int(self->input_zero_point);
+                dest[0] = MP_OBJ_FROM_PTR(self->input_zero_point);
                 break;
             case MP_QSTR_output_shape:
                 dest[0] = MP_OBJ_FROM_PTR(self->output_shape);
                 break;
             case MP_QSTR_output_dtype:
-                dest[0] = mp_obj_new_str(&self->output_dtype, 1);
```

Comment:
This used to return a nice character type, now it will return a tuple of digits.

Reply:
Vectorized this. :)